.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
```

Comment:
This file doesn't seem to be running nor correctly implemented with all the env variables necessary to run the tests.

Reply:
fixed